trin-core/src/utp/packets.rs

```diff
@@ -142,7 +142,7 @@ impl PacketHeader {
 
     /// Returns the packet's type.
     pub fn get_type(&self) -> PacketType {
-        PacketType::try_from(self.type_ver >> 4).unwrap()
+        PacketType::try_from(self.type_ver >> 4).expect("Invalid packet type")
```

Comment:
This string is used already. For uniqueness:

```suggestion
        PacketType::try_from(self.type_ver >> 4).expect("Can't parse packet type")
```